tests/cli-inst-interactive.rs

```diff
@@ -254,3 +254,21 @@ fn install_forces_and_skips_rls() {
             .contains("warning: Force-skipping unavailable component"));
     });
 }
+
+#[test]
+fn test_warn_if_complete_profile_is_used() {
+    setup(&|config| {
+        expect_err(
+            config,
+            &[
+                "rustup",
+                "toolchain",
+                "install",
+                "--profile",
+                "complete",
+                "stable",
+            ],
+            "warning: downloading with complete profile isn't recommended unless you are a developer of the rust language",
+        );
+    });
+}
```

Comment:
The `cli-inst-interactive.rs` test file is really for `rustup-init` testing.  This test belongs more in `cli-v2.rs` or similar where you already have it.  As such this should probably be to test `rustup-init --profile complete --default-toolchain stable -y` or somesuch.  (See other tests in this file)

Reply:
I already changed that